test/functional/apps/discover/_filter_editor.ts

```diff
@@ -16,7 +16,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
   const kibanaServer = getService('kibanaServer');
   const filterBar = getService('filterBar');
-  const PageObjects = getPageObjects(['common', 'discover', 'timePicker']);
```

Comment:
I think this could be e a typical cloud testing error, maybe insufficient permission:
https://github.com/elastic/kibana/pull/126377 adding `version_test`
But it doesn't seem to be used like `context_encoded_param` here

https://github.com/elastic/kibana/blob/553374816224ac901327c9dae5e75b8ffa70b79e/test/functional/apps/discover/_context_encoded_url_param.ts#L22

```TypeScript
await security.testUser.setRoles(['kibana_admin', 'version_test']);
```